src/components/CurrentWalletMagic.jsx

```diff
@@ -83,10 +83,10 @@ export default function CurrentWalletMagic() {
       .catch((err) => {
         !abortCtrl.signal.aborted && setAlert({ variant: 'danger', message: err.message })
       })
-      .finally(() => setIsLoading(false))
+      .finally(() => !abortCtrl.signal.aborted && setIsLoading(false))
```

Comment:
Ah I see, that caused the "unmounted component state change" warnings, right?

Reply:
Yep, think we still have a few of them. Low priority, but we'll become more and more aware in code reviews, I guess.